core/src/main/java/com/yahoo/ycsb/generator/ExponentialGenerator.java

```diff
@@ -80,9 +67,10 @@ public int nextInt()
 	 * @param itemcount The number of items in the distribution.
```

Comment:
Looks like this param `itemcount` isn't valid anymore (and wasn't valid before?)